lib/index.js

```diff
@@ -49,7 +49,8 @@ function createTouchList(target, list) {
         list = [list];
     }
     list = list.map(function (entry, index) {
-        return createTouch(entry[0], entry[1], target, index + 1);
+        var x = entry[0], y = entry[1], id = entry[2] ? entry[2] : index + 1;
```

Comment:
Thanx.

Code style: If you could modify the line 52 to have one var statement for each variable - that would be great and merged :).